packages/sdk-auth/test/token-provider.spec.js

```diff
@@ -41,6 +41,34 @@ describe('Token Provider', () => {
       )
     })
 
+    test('should clear cache after _performFetchTokenInfo rejects', async () => {
+        const _tokenProvider = new TokenProvider({
+            sdkAuth,
+            fetchTokenInfo: jest
+                .fn()
+                .mockImplementation(() => Promise.reject(new Error("Invalid Token"))),
+        })
+        expect(_tokenProvider.fetchTokenInfoPromise).toBeFalsy();
+        try {
+            await _tokenProvider._performFetchTokenInfo()
+        } catch(e) { /** */}
```

Comment:
and here I would recommend wrapping the below code inside the catch block instead of commenting it out. IMO it would make it more clear.

Reply:
Done